.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseTimePeriodExtractor.cs

```diff
@@ -153,6 +154,22 @@ private List<Token> MatchSimpleCases(string text)
             return ret;
         }
 
+        // Cases like "from 6am for 3 hours" and "for 3 hours from 6 am" are extracted as timerange here.
+        private List<Token> MatchTimePeriodWithDurationCases(string text)

```

Comment:
Is this code being used by an english implementation class ?

Reply:
As we talked, will consider it in next sprint.